package-lock.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
What's up with this file? Do we want it? If we do, I think we should `COPY` it in the dockerfile and document how to update it over time

Reply:
I'm including it for now because `npm install` outputs a warning when it writes the file. We'll be removing this particular `npm install` when we move over to the parser soon, so I'm inclined to punt on the documentation. Added that `COPY` argument though :+1: